packages/commerce-sdk-react/README.md

```diff
@@ -64,7 +101,259 @@ const App = ({children}) => {
+
+### Shopper Session Initialization
+
+On `CommerceApiProvider` mount, the provider initializes shopper session by initiating the SLAS Public Client login flow. The tokens are stored/managed by the library to support Phased Launch use cases and multi-site setup.
+
+When the access token is expired, the library will automatically try to refresh the access token by calling the `/oauth2/token` endpoint with `grant_type=refresh_token`.
+
+### Authenticate request queue
+
+While the library is getting/refreshing the access token, the network requests will queue up until there is a valid access token.
+
+### Auth helpers
+
+To leverage the managed shopper authentication feature, use the auth helper hooks for shopper login.
+
+Example:
+
+```jsx
+import {AuthHelpers, useAuthHelper} from '@salesforce/commerce-sdk-react'
+
+const Example = () => {
+    const register = useAuthHelper(AuthHelpers.Register)
+    const login = useAuthHelper(AuthHelpers.LoginRegisteredUserB2C)
+    const logout = useAuthHelper(AuthHelpers.LogOut)
+
+    return <button onClick={() => {
+        login.mutate({username, password})
+    }}>
+}
+```
+
+### Opt-out from managed shopper authentication
+
+To opt-out of the auto-login, request queue, and token management features, pass `fetchedToken` (a SLAS JWT) to the provider.
+
+```jsx
+const MyComponent = ({children}) => {
+    return (
+        <CommerceApiProvider
+            fetchedToken="xxxxxxxxxxxx"
+        >
+            {children}
+        </CommerceApiProvider>
+    )
+} 
+```
+
+## Queries
+
+The query hooks correspond to the http GET endpoints from the SCAPI family. The query hooks follow the signature pattern:
+
+```
+use<EntityName>(CommerceClientOptions, ReactQueryOptions)
+```
+
+For example, both the __required__ and __optional__ parameters for the underlying `commerce-sdk-isomorphic` call is passed as the first parameter:
+
+```jsx
+import {useProduct} from '@salesforce/commerce-sdk-react'
+
+const Example = () => {
+    const query = useProduct({
+        parameters: {
+            id: '25592770M',
+            locale: 'en-US'
+        }
+    })
+
+    return <>
+        <p>isLoading: {query.isLoading}</p>
+        <p>name: {query.data?.name}</p>
+    </>
+}
+```
+
+The second parameter is the react-query query options, for more detail, read [useQuery reference](https://tanstack.com/query/latest/docs/react/reference/useQuery).
+
+```jsx
+import {useBasket} from '@salesforce/commerce-sdk-react'
+
+const onServer = typeof window === undefined
+
+const Example = ({basketId}) => {
+    const query = useBasket({
+        parameters: {
+            basketId: basketId
+        },
+    }, {
+        // A common use case for `enabled` is
+        // to conditionally fetch based on environment
+        enabled: !onServer && basketId
+    })
+}
+```
+
+### Advanced: Query Key Structure
```

Comment:
If the answer to the question "Is our key structure private?" is "Yes" then I would leave this entire section out. I feel its going to be difficult to give a complete picture of the entire cache structure in this doc, and those wanting to do advanced things will be using the chrome dev tools and they can sus it out from there. Maybe we need an example of using those dev tools?

Reply:
removed